src/main/python/systemml/converters.py

```diff
@@ -212,10 +215,38 @@ def convertToNumPyArr(sc, mb):
     else:
         raise TypeError('sc needs to be of type SparkContext') # TODO: We can generalize this by creating py4j gateway ourselves
 
+# Returns the mean of a model if defined otherwise None
+def getModelMean(model_name):
+    """
+    Input Parameters
+    ----------------
+    model_name: Name of the model
+
+    Returns
+    -------
+    mean: Mean value of model if its defined in the list MODEL_MEAN else None.
+
+    """
+
+    try:
+        mean = MODEL_MEAN[model_name.upper()]
+    except:
+        mean = None
+    return mean
+
+
 # Example usage: convertImageToNumPyArr(im, img_shape=(3, 224, 224), add_rotated_images=True, add_mirrored_images=True)
 # The above call returns a numpy array of shape (6, 50176) in NCHW format
-def convertImageToNumPyArr(im, img_shape=None, add_rotated_images=False, add_mirrored_images=False):
-    from PIL import Image
+def convertImageToNumPyArr(im, img_shape=None, add_rotated_images=False, add_mirrored_images=False,
+    color_mode = 'RGB', mean=None):
+
+    ## Input Parameters
+
+    # color_mode: In case of VGG models which expect image data in BGR format instead of RGB for other most models,
+    # color_mode parameter is used to process image data in BGR format.
+
+    # mean: mean value is used to subtract from input data from every pixel value. Default value specified is for VGG-19 model.
```

Comment:
can you please update the documentation to specify that by default we donot subtract mean ?